source/glest_game/global/core_data.cpp

```diff
@@ -62,14 +62,13 @@ namespace Glest {
 		static const string CORE_WATER_SOUNDS_PATH = CORE_PATH + "/water_sounds/";
 
 		// Sound effects
-		// These variables are specified in the ini file
-		const string PlaySoundClip::sfxAttention = "PlaySoundAttention";
-		const string PlaySoundClip::sfxHighlight = "PlaySoundHighlight";
-		const string PlaySoundClip::sfxNewServer = "PlaySoundNewServer";
-		const string PlaySoundClip::sfxMarker = "PlaySoundMarker";
-		const string PlaySoundClip::sfxMenuClickA = "PlaySoundMenuClickA";
-		const string PlaySoundClip::sfxMenuClickB = "PlaySoundMenuClickB";
-		const string PlaySoundClip::sfxMenuClickC = "PlaySoundMenuClickC";
+		const string sfxAttention = data_path + CORE_MENU_SOUND_PATH + "attention.wav";
```

Comment:
@mathusummut , data_path is used elsewhere but it's not declared here yet. I'm not sure where to put these 7 assignments.